marklogic-data-hub-central/src/test/java/com/marklogic/hub/central/AbstractHubCentralTest.java

```diff
@@ -127,8 +128,8 @@ protected File getTestProjectDirectory() {
     @Override
     protected HubClient runAsUser(String username, String password) {
         // Initialize a new HubConfigImpl
-        testHubConfig = hubCentral.newHubConfig(username, password);
-        testHubConfig.setHubProject(testHubProject);
+        testHubConfig.createProject(testProjectDirectory);
```

Comment:
The comment above this should be removed now - and really, we ought to explain why the project is being created here. So I think a comment of "Need to create the project directory before applying properties" (assuming that is the correct reason).